docker-compose.yml

```diff
@@ -19,15 +14,55 @@ services:
     ports:
       - "3333:3333"
     environment:
-      - "REDIS_URL=${REDIS_URL-redis://grant-redis:6379}"
+      - "DATABASE_URL=postgres://grants:password@postgres/grants?sslmode=disable"
+      - "LEDGER_SERVER=https://ledger-staging.mercury.basicattentiontoken.org"
+      - "REDIS_URL=${REDIS_URL-redis://redis:6379}"
```

Comment:
does it make sense to remove the option to swap this out. just to avoid situations like those that we saw with ledger.

Reply:
good idea - let's avoid that